### ISSUE TITLE
Upgrade kotlin dependency to 1.2.60

### ISSUE THREAD

**Issue by bamboo** [gradle/kotlin-dsl#940]
Upgrading to 1.2.50 is blocked by [KT-24946](https://youtrack.jetbrains.com/issue/KT-24946).

**Comment by bamboo** [comment]
Upgrade to Kotlin 1.2.60-eap-44. Follow-up tasks will be handled as separate issue.
